src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/mapper/OffenderToAssessmentSummaryMapper.kt

```diff
@@ -0,0 +1,44 @@
+package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.mapper
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.entity.Offender
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.exception.PrisonerNotFoundException
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.model.AssessmentSummary
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.model.TaskProgress
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.model.toSummary
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.prison.PrisonService
+
+@Component
+class OffenderToAssessmentSummaryMapper(
+  private val prisonService: PrisonService,
+) {
+
+  fun map(offender: Offender): AssessmentSummary {
+    val currentAssessment = offender.currentAssessment()
+    val prisonerSearchResults = prisonService.searchPrisonersByNomisIds(listOf(offender.prisonNumber))
+    if (prisonerSearchResults.isEmpty()) {
+      throw PrisonerNotFoundException(offender.prisonNumber)
+    }
+    val prisonerDetails = prisonerSearchResults.first()
+    return AssessmentSummary(
+      forename = offender.forename,
+      surname = offender.surname,
+      dateOfBirth = offender.dateOfBirth,
+      prisonNumber = offender.prisonNumber,
+      hdced = offender.hdced,
+      crd = offender.crd,
+      location = prisonerDetails.prisonName,
```

Comment:
Making assumptions that the prison name will be correct from the prisoner details query.
Could be problems when the prisoner is being temporary moved, but this should not be a problem at this stage in the sentence.

Reply:
Could we grab this as previously so it's consistent with the current assessment view?
 `val prisonName = prisonService.getPrisonNameForId(offender.prisonId)`

(we handle transfers separately)

Reply:
Yes no probs